commit b43d8aaa089caf92d9d2d6b94f0dae3e907e940b
Author: maxwen <max.weninger@gmail.com>
Date:   Fri Dec 8 14:08:26 2017 +0100

    Revert "Remove duplicate targets."
    
    This reverts commit 4f4c5a6450d23be03234069a4cf8fe2cd23a0965.
    
    Change-Id: I13998e768fb19e6ed4c9d2dd6ac10bc82da48bc1

diff --git a/transport/base/1.0/Android.bp b/transport/base/1.0/Android.bp
index bd2ae49..8d8e38d 100644
--- a/transport/base/1.0/Android.bp
+++ b/transport/base/1.0/Android.bp
@@ -17,3 +17,24 @@ hidl_interface {
     gen_java: true,
 }
 
+cc_library_shared {
+    name: "android.hidl.base@1.0",
+    generated_sources: ["android.hidl.base@1.0_genc++"],
+    generated_headers: ["android.hidl.base@1.0_genc++_headers"],
+    export_generated_headers: ["android.hidl.base@1.0_genc++_headers"],
+    vendor_available: true,
+    shared_libs: [
+        "libhidlbase",
+        "libhidltransport",
+        "libhwbinder",
+        "liblog",
+        "libutils",
+        "libcutils",
+    ],
+    export_shared_lib_headers: [
+        "libhidlbase",
+        "libhidltransport",
+        "libhwbinder",
+        "libutils",
+    ],
+}
diff --git a/transport/manager/1.0/Android.bp b/transport/manager/1.0/Android.bp
index e4a120b..59abc13 100644
--- a/transport/manager/1.0/Android.bp
+++ b/transport/manager/1.0/Android.bp
@@ -17,3 +17,24 @@ hidl_interface {
     gen_java: true,
 }
 
+cc_library_shared {
+    name: "android.hidl.manager@1.0",
+    generated_sources: ["android.hidl.manager@1.0_genc++"],
+    generated_headers: ["android.hidl.manager@1.0_genc++_headers"],
+    export_generated_headers: ["android.hidl.manager@1.0_genc++_headers"],
+    vendor_available: true,
+    shared_libs: [
+        "libhidlbase",
+        "libhidltransport",
+        "libhwbinder",
+        "liblog",
+        "libutils",
+        "libcutils",
+    ],
+    export_shared_lib_headers: [
+        "libhidlbase",
+        "libhidltransport",
+        "libhwbinder",
+        "libutils",
+    ],
+}
